Atomic/syscontainers.py

```diff
@@ -286,7 +286,8 @@ def _gather_installed_files_info (checkout, name):
 
         return info, rpm_installed, installed_files_checksum
 
-    def _get_remote_location(self, remote_input):
+    @staticmethod
+    def _get_remote_location(remote_input):
```

Comment:
I think it should be a static method as there is no `self` involved in the function, but  I normally made a decision of "static vs non-static" according to my instinct. So, is there a common way to identify when to make a function as static and when not to? (/me also searches for this online). Any suggestions are welcome =).